### PR TITLE
task Catalog

### PR DESCRIPTION
**Pull request by vladgusakovskiy**
- [DEMO LINK](https://vladgusakovskiy.github.io/layout_catalog/)
- [TEST REPORT LINK](https://vladgusakovskiy.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Comment by adrqq** [comment]
great job)

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,509 @@
...
+                Видео
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="card-container">
+        <section class="card" data-qa="card">
```
Comment (dellachaise) [inline src/index.html:68]: I see no styles for `main` class. You could delete it. Better name for `card-container` - `catalog`. And you could use mix. 
 More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

```suggestion
    <main>
      <div class="catalog">
        <section class="card catalog__card" data-qa="card">
```

File: src/styles/blocks/cards.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  box-sizing: border-box;
+
+  list-style: none;
+  border-radius: $border-radius;
+  border: 1px solid #f3f3f3;
+
+  &-container {
+    display: flex;
+    flex-wrap: wrap;
+    width: 944px;
```
Comment (dellachaise) [inline src/styles/blocks/cards.scss:15]: Do not hardcode width, calculate it.

File: src/styles/blocks/cards.scss [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    padding: 50px 40px;
+    margin: 0 auto;
+  }
+
```
Comment (dellachaise) [inline src/styles/blocks/cards.scss:19]: it's not `card` element or modifier. It's a separate block witch use `-` in name consisted from 2 words.
Move this code to a separate file.
Rewatch info about naming in [this video](https://mate.academy/learn/css-advanced/bem#/video/920)

**Review by dellachaise — CHANGES_REQUESTED** [review]
Great start 👍

Please, add fixes according to comments.

File: src/styles/blocks/catalog.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+
+  width: $catalog-width;
+  margin: 0 auto;
```
Comment (dellachaise) [inline src/styles/blocks/catalog.scss:7]: You can't set margin to block, only for elements. You can use mix(set `page` class for body and add class `page__catalog` and move margin there.

 More details in video.
[video, timing 4:15] (https://mate.academy/learn/css-advanced/bem#/video/911)
Screenshot 2022-07-21 at 19 28 44
(https://mate.academy/learn/css-advanced/bem#/video/911)

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
+.header {
+  margin: 0 auto;
```
Comment (dellachaise) [inline src/styles/blocks/header.scss:2]: Also, move margin to `.page__header`. More details in previous comment.

File: src/styles/blocks/stars.scss [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 > &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```
Comment (dellachaise) [inline src/styles/blocks/stars.scss:16]: You should have possibility to change amount of active stars by changing a modifier. You could use code from `stars` task. 

And try to use cycle for
more detail in [video](https://mate.academy/learn/css-advanced/sass#/video/926)

**Review by dellachaise — CHANGES_REQUESTED** [review]
Great progress 👍 

Please, add fixes according to last for this task comments.

**Review by dellachaise — APPROVED** [review]
Well done 👍  🔥

**Pull request closed without merging**